x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/action/CrossClusterQueriesWithInvalidLicenseIT.java

```diff
@@ -65,25 +66,30 @@ public void testQueryAgainstNonMatchingClusterWildcardPattern() {
 
         // since this wildcarded expression does not resolve to a valid remote cluster, it is not considered
         // a cross-cluster search and thus should not throw a license error
-        String q = "FROM xremote*:events";
-        {
-            String limit1 = q + " | STATS count(*)";
-            try (EsqlQueryResponse resp = runQuery(limit1, requestIncludeMeta)) {
-                assertThat(resp.columns().size(), equalTo(1));
-                EsqlExecutionInfo executionInfo = resp.getExecutionInfo();
-                assertThat(executionInfo.isCrossClusterSearch(), is(false));
-                assertThat(executionInfo.includeCCSMetadata(), equalTo(responseExpectMeta));
-            }
-
-            String limit0 = q + " | LIMIT 0";
-            try (EsqlQueryResponse resp = runQuery(limit0, requestIncludeMeta)) {
-                assertThat(resp.columns().size(), equalTo(1));
-                assertThat(getValuesList(resp).size(), equalTo(0));
-                EsqlExecutionInfo executionInfo = resp.getExecutionInfo();
-                assertThat(executionInfo.isCrossClusterSearch(), is(false));
-                assertThat(executionInfo.includeCCSMetadata(), equalTo(responseExpectMeta));
-            }
-        }
+        expectThrows(
+            VerificationException.class,
+            containsString("Unknown index [xremote*:events]"),
+            () -> runQuery("FROM xremote*:events | STATS count(*)", requestIncludeMeta).close()
+        );
```

Comment:
Prior behavior returned empty result for empty index resolution.
Now this is complaining about empty index resolution (consistent with `FROM empty-pattern-*` case)